src/Integration.Vsix/Commands/PackageCommandId.cs

```diff
@@ -31,6 +31,7 @@ internal enum PackageCommandId
         // Menus
         ProjectSonarLintMenu = 0x850,
         // Help menu buttons
-        SonarLintHelpShowLogs = 0x102
+        SonarLintHelpShowLogs = 0x102,
+        ViewDocumentationCommand = 0x103
```

Comment:
I just noticed that we have a problem. This enum doesn't take into account the fact that it's the same ID but under a different set. 0x102 and 0x103 are already taken by the commands above, and I think we created a conflict here. 

I suggest that we place these IDs in the relevant command classes:
```
internal class ViewDocumentationCommand : VsCommandBase
{
 internal const int ViewDocumentationCommandId = 0x103;
}
```

WDYT?

Reply:
but then how would it look in the packageCommandManager.cs? Like this?

```
var viewDocumentationCommand = new ViewDocumentationCommand();
this.RegisterCommand(CommonGuids.HelpMenuCommandSet, viewDocumentationCommand.ViewDocumentationCommandId, viewDocumentationCommand);
```